tool/generate_local_bundles.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Runs `mason bundle` to generate bundles for all bricks within the top level bricks directory.
```

Comment:
Avoid to generate bundles locally because https://github.com/felangel/mason/issues/781